app/src/components/page/Sidebar/index.tsx

```diff
@@ -96,7 +97,11 @@ export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) =>
         className="menu-dark"
       >
         {ENABLE_PLANS && (
-          <Menu.SubMenu key="missions" icon={<DashboardOutlined />} title={MISSIONS}>
+          <Menu.SubMenu
+            key="missions"
+            icon={<MapMarkerOutline className="sidebar-icons" />}
```

Comment:
Why did we change the sizes?

Reply:
they were relatively small in size, their current size is as shown in the above screenshot